src/test/java/org/datadog/jenkins/plugins/datadog/publishers/DatadogQueuePipelinePublisherTest.java

```diff
@@ -36,7 +36,7 @@ public void testPipelineInQueue() throws Exception {
 
         // schedule build and wait for it to get queued
         QueueTaskFuture<WorkflowRun> task = job.scheduleBuild2(0);
-        Thread.sleep(1000);
+        Thread.sleep(10000);
```

Comment:
Could we use 1 sec?

Reply:
1s is flaky for the test

Reply:
Have you tried with fewer seconds? 10 seconds look too much, IMO

Reply:
could you please suggest a reasonable number? I can just update to that number

Reply:
I'd try with 2 seconds if 1 does not work